src/scripts/http/cachedHttp.ts

```diff
@@ -68,19 +63,17 @@ export class CachedHttp {
 			throw new Error("getRemoteValue must be non-undefined");
 		}
 
-		return new Promise<TimeStampedData>((resolve, reject) => {
-			getRemoteValue().then((responsePackage) => {
-				let setValue: TimeStampedData = this.setTimeStampedValue(key, responsePackage.parsedResponse);
-				if (!setValue) {
-					// Fresh data from the remote was unavailable
-					this.cache.removeKey(key);
-				}
-				resolve(setValue);
-			}, (error: OneNoteApi.RequestError) => {
-				// TODO: Don't use OneNoteApi.RequestError
+		return getRemoteValue().then((responsePackage) => {
```

Comment:
getRemoteValue is passed in as a parameter and defines how the remove value should be gotten.